### DISCUSSION TITLE
remove constraints from a relation field

### DISCUSSION THREAD

**Discussion by CACG1022** [opengisch/QField#4087] [Q&A]
Is there any way to remove the constraints from a relation field? since I need to first enter data to a child form and then continue with the filling of mandatory fields in the main form?
![ayuda5](https://user-images.githubusercontent.com/118937011/228310197-4566e777-e546-409f-aeee-29dca9eb8a27.png)

**Comment by lucienicolier** [comment]
Hi @CACG1022,

To set up constraint, you will need to do it on QGIS.

Have a look at the doc [Define Constraints](https://docs.qfield.org/how-to/attributes-form/?h=constr#define-constraints)
